### PR TITLE
Eliminate the hardcoded AmsterdamLogo from the Header

### PR DESCRIPTION
**Pull request by aditudorache**
This PR:
- Addresses Issue #227
- removes the hard-coded AmsterdamLogo component
- this is a *breaking change* the existing sites should provide the AmsterdamLogo as parameter of the Header component

### CODE REVIEW

**Comment by jonkoops** [comment]
Gave this PR a quick rebase to ensure it's in line with the master branch.

**Comment by jonkoops** [comment]
@chrisvanmook Can you give this PR a review as well?

File: packages/asc-ui/src/components/AmsterdamLogo/AmsterdamLogoStyle.ts [hunk 1]
```diff
@@ -19,7 +19,8 @@ const tallStyle = css`
   @media screen and ${breakpoint('min-width', 'laptopM')} {
     height: 68px;
     width: 100px;
-
+    margin-right: 40px;
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/AmsterdamLogo/AmsterdamLogoStyle.ts]: can we use `themeSpacing(10)` here?

File: packages/asc-ui/src/components/AmsterdamLogo/AmsterdamLogoStyle.ts [hunk 2]
```diff
@@ -31,6 +32,7 @@ const AmsterdamLogoStyle = styled.a<Props>`
   height: 30px;
   width: 68px;
   flex-shrink: 0;
+  margin-right: 10px;
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/AmsterdamLogo/AmsterdamLogoStyle.ts]: same, `themeSpacing(2)` or perhaps 3

File: packages/asc-ui/src/components/Header/Header.stories.tsx [hunk 3]
```diff
@@ -1,7 +1,28 @@
...
+import AmsterdamLogo from '../AmsterdamLogo'
+import Icon from '../Icon'
+import { LogoProps } from './HeaderLogoText'
+
+/**
+ * This is an example of a Custom facebook logo
+ * that can be added to the Header
+ */
+const CustomLogoStyle = styled.a`
+  margin-right: 10px;
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Header/Header.stories.tsx]: same, themespacing

File: CHANGELOG.md [hunk 4]
```diff
@@ -16,7 +16,10 @@ Prefix the change with one of these keywords:
 
 ## Unreleased
 
+- Changed: **BREAKING** The AmsterdamLogo is not hardcoded anumore in the header. Has to be passed as parameter.
+
 ## [0.18.2]
+
```
[the remark this answers is not in the document]
Comment (jonkoops) [reply CHANGELOG.md]: We're not using markdown lint though and Github flavored markdown specifies nothing of this.

**Pull request merged**